lime/tools/platforms/WindowsPlatform.hx

```diff
@@ -141,12 +155,21 @@ class WindowsPlatform extends PlatformTarget {
 				
 			}
 			
+			if (enableLinuxMingw) {
```

Comment:
These are compile flags needed when compiling Win32 application from Linux (Fedora and Ubuntu).